runfolder/lib/instrument.py

```diff
@@ -33,6 +33,8 @@ def get_instrument(run_parameters):
 
         if re.search(NovaSeq.ID_PATTERN, instrument_id):
             return NovaSeq()
+        if re.search(NovaSeqXPlus.ID_PATTERN, instrument_id):
```

Comment:
Instrument ID for NovaSeq X Plus is found in key "InstrumentSerialNumber" in RunParameters.xml. Can you add that to the list on line 20?